src/test/java/database/MySQLDatabaseGatewayTests.java

```diff
@@ -1,8 +1,13 @@
 package database;
 
-import org.junit.*;
-import static org.junit.Assert.*;
-import billgates.database.*;
+import billgates.Main;
+import billgates.database.MySQLDatabaseGateway;
+import billgates.database.QueryBillData;
+import billgates.database.QueryEntryData;
```

Comment:
With the pull request I attempted [here](https://github.com/CSC207-2022F-UofT/course-project-bill-gates/pull/76)

I believe we should remove the classes `QueryBillData` and `QueryEntryData`. 
We can fix that when solving conflicts in merges